libimage/image.go

```diff
@@ -328,7 +347,7 @@ func (i *Image) Untag(name string) error {
 	}
 
 	if !removedName {
-		return nil
+		return errors.Errorf("%q: no such tag", name)
```

Comment:
This should probably be a constant.

Reply:
I _really_ do not liked exported errors unless they are absolutely necessary.

Reply:
You don't think someone at some point might want to check for TagNoExists versus some other error?

Reply:
At the moment, we don't need it.  There's just one test looking for the error message.

I wrote down a number of lessons learned from that work, one of them is touching that aspect.  As I read and touched so many lines of code, I noticed that exposing too many internals to outside users is a common pattern for (too) complex code at the users/callers.

The most extreme example I faced was the `rmi` code in Podman which was scattered over three packages, each adding a new layer of logic to it.

So far, `libimage` does not add any new error to it but forwards the one from `c/storage`.